src/components/Settings/AppearanceSettings.tsx

```diff
@@ -15,14 +15,14 @@ export function AppearanceSettings() {
   return (
     <SettingsSection>
       <FieldGroup
-        name="appearance.theme"
+        name="general.appearance.theme"
```

Comment:
This change is intended for testing purposes and will be reverted before merging.